tests/python_tests/test_vlm_pipeline.py

```diff
@@ -55,14 +55,16 @@ def streamer(word: str) -> bool:
         return False
 
     models_path = get_ov_model(model_id, cache)
-    generation_config = GenerationConfig(max_new_tokens=30)
 
     for links in image_links_for_testing:
         images = []
         for link in links:
             images.append(get_image_by_link(link))
 
         ov_pipe = VLMPipeline(models_path, "CPU")
```

Comment:
why do we need to load model each time?
we can configure pipeline before a loop